R/translate.R

```diff
@@ -60,15 +60,33 @@ deepl_translate <- function(path,
     formality,
     values = c("default", "more", "less", "prefer_more", "prefer_less")
   )
-
-  # protect Hugo shortcodes ----
   temp_markdown_file <- withr::local_tempfile()
   markdown_lines <- brio::read_lines(path)
+  temp_markdown_file <- withr::local_tempfile()
+
+  # protect $$ equations $$
+  dollars <- which(grepl("\\$\\$", markdown_lines))
+  dollars_present <- length(dollars > 0)
```

Comment:
should there also be a check of the number being even? this would simplify the code below.